frontend/src/store/features/airdrop/airdropService.js

```diff
@@ -0,0 +1,28 @@
+import Axios from 'axios';
+import { cleanURL } from '../utils';
+
+const claimRecordsURL = '/passage3d/claim/v1beta1/claim_record/';
+const claimParamsURL = '/passage3d/claim/v1beta1/params';
+
+const getClaimRecords = (baseURL, address) => {
+  const uri = `${cleanURL(baseURL)}${claimRecordsURL}${address}`
+  return Axios.get(uri)
+}
+
+const getAirdropDetails = (address) => {
+  const uri = `https://api.resolute.vitwit.com/passage-airdrop-details/airdrop-check?address=${address}`
```

Comment:
It is recommended to define constants and use them in your code instead of directly using their values. This will make your code more readable, easier to maintain and update, and less prone to errors.